data/projects/implement_metrics_for_coala.md

```diff
@@ -7,28 +7,46 @@ them can be measured objectively by so-called
 offers a good base for the implementation or wrapping of algorithms that
 calculate these metrics from a piece of code.
 
+coala should have a module that can produce software metrics because software
+metrics are crucial when it comes to quality assurance. Source code metrics
+for quality assurance aims at writing code with less bugs inside and writing
+code for easy maintenance. Both lead to writing code as simple as possible
+with few elements in each unit. Picking right metrics is paramount and this
+project aims at implementing an API and a set of Bears for producing
+software metrics.
+
 The goal of this project proposal should be to implement handling and
 display of software metrics in coala. To prove the concept, this project
 has to include multiple bears that calculate metrics for different
 programming languages.
 
+
 #### Milestones
 
-##### GSOC 2017 COMMUNITY BONDING
+##### PREPARATION/BONDING
 
+* Research existing tools for software metrics.
 * The applicant has created CLI and web UI mockups for metrics.
 * The applicant has created a [cEPs](https://coala.io/cep) for his project.
 
-##### GSOC 2017 MIDTERM
+##### CODING PHASE 1
 
-* The API for bears to yield metrics has been implemented.
-* The API is properly documented and fully tested.
+* The API for bears to yield metrics is done and well documented.
 * At least one Bear implementing the API exists.
 * A working (not clean or merged!) prototype of the CLI visualization
 exists.
 
-##### GSOC 2017 FINAL
+##### CODING PHASE 2
+
+* The API for bears to yield metrics is merged into coala.
```

Comment:
this is kind of the same as the first one. I'd put the completion of api concept in the preparation, implementation in first phase and then 1 bear in first phase, UI stuff in second phase more bears and tutorial 3rd phase. All code has to be tested and documented when it's merged so the "testing has begun" milestone is wrong as it has to be done together with the implementation